client/src/components/Pages/Result/index.tsx

```diff
@@ -22,7 +22,7 @@ export const Result: React.FunctionComponent = () =>{
   const [ended, setEnded] = useState(false);
   let history = useHistory();
   const { oidcUser } = useReactOidc();
-  const evals = oidcUser.profile.groups.includes("eboard-evaluations");
+  const evals = true oidcUser.profile.groups.includes("eboard-evaluations");
```

Comment:
evals=true should be removed here before merge also

Reply:
Fuck. I knew I'd miss at least one.